### PR TITLE
Update dragmap + add nf-test

### PR DESCRIPTION
**Pull request by nvnieuwk**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/dragmap/align/environment.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ channels:
   - bioconda
   - defaults
 dependencies:
-  - dragmap=1.2.1
+  - dragmap=1.3.0
   # renovate: datasource=conda depName=bioconda/samtools
-  - samtools=1.15.1
+  - samtools=1.16
```
[the remark this answers is not in the document]
Comment (nvnieuwk) [reply modules/nf-core/dragmap/align/environment.yml:9]: It's 1.16 in the mulled container, so I'd rather keep them the same

[the remark this answers is not in the document]
Comment (FriederikeHanssen) [reply modules/nf-core/dragmap/align/environment.yml:9]: in that case maybe update the mulled container as well? but in any case 1.3.0 has a bunch of issues to my knowledge https://github.com/Illumina/DRAGMAP/issues/47

[the remark this answers is not in the document]
Comment (nvnieuwk) [reply modules/nf-core/dragmap/align/environment.yml:9]: https://github.com/BioContainers/multi-package-containers/pull/2839

[the remark this answers is not in the document]
Comment (Emiller88) [reply modules/nf-core/dragmap/align/environment.yml]: I also have the 1.3.0 bug, so I don't know if we can move forward.

https://github.com/Illumina/DRAGMAP/commit/c4cb6c94baaf6821db3fa696c53cdd809fb92304

It looks like there's a 1.3.1 but they didn't tag it properly, so bioconda didn't create a release.

[the remark this answers is not in the document]
Comment (Emiller88) [reply modules/nf-core/dragmap/align/environment.yml]: It's GPL3, do we fork it? 😆 There's a lot of improvement PRs

[the remark this answers is not in the document]
Comment (nvnieuwk) [reply modules/nf-core/dragmap/align/environment.yml]: Good for me :p (And I know the tests are failing, I still need to do a small fix)

**Comment by nvnieuwk** [comment]
I'd rather wait for your updates then so we're sure it works as intended :grin:

**Comment by Emiller88** [comment]
I think that'll be a whole new module. I think this is as far as dragmap is gonna get, they never tagged the 1.3.1 release. So we might as well merge this?

**Comment by nvnieuwk** [comment]
Fair enough :shrug:

**Pull request merged**